shaky/src/main/res/values/shaky_attrs.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  Copyright (C) 2016 LinkedIn Corp.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+          http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<resources>
+    <!-- Common -->
+    <attr name="shakyBackgroundColor" format="reference|color" />
+    <attr name="shakyTitleColor" format="reference|color" />
+    <attr name="shakyContentColor" format="reference|color" />
+    <attr name="shakyToolbarColor" format="reference|color" />
+    <attr name="shakyToolbarTitleTextColor" format="reference|color" />
```

Comment:
The entire toolbar style / theme should be an attribute, and we should just let users set the style on the toolbar which in turn defines the text color, background color, elevation etc.